### DISCUSSION TITLE
Multiple pseudo locales

### DISCUSSION THREAD

**Discussion by matzeeable** [WeblateOrg/weblate#6487] [Q&A] [answered]
Hi there!

Is it somehow possible to use pseudo locales for multiple languages? I do only translate `de_DE` but want to have pseudo locales for `de_AT` and `de_CH`. I can only configure one pseudo locale for a component.

**Comment by nijel** [comment, marked as answer]
Do you mean [Pseudolocale generation add-on](https://docs.weblate.org/en/latest/admin/addons.html#pseudolocale-generation)? Installing that multiple time will be supported in 4.8.1 (see 053d395e93).

**Reply by matzeeable** [reply]
Awesome, thanks!
